src/server/acl.json

```diff
@@ -0,0 +1,61 @@
+{
+  "supplier-mgmt": {
+    "translations": {
+      "name" : {
+        "en": "Suppplier Management for supplier-tenant",
+        "de": "Lieferantenmanagement für supplier-tenant"
+      },
+      "description": {
+        "en": "Suppplier management includes all administrative functions for supplier management, such as supplier modification, contact, address and bank account creation, edit, and deletion.",
+        "de": "Das Lieferantenmanagement umfasst alle administrativen Funktionen für das Lieferantenmanagement, wie zB Lieferantenänderung, Kontakt-, Adress- und Bankkonto-Erstellung, Bearbeiten und Löschen."
+      }
+    },
+    "resources": [
+      {
+        "type": "rest",
+        "resourceId": "/api/suppliers[/]?$",
+        "actions": ["create"]
```

Comment:
careful, this is required fro supplier registration, but the create supplier may only happen in case user is not yet assigned to supplier